packages/backend/app/Services/OrderServices.ts

```diff
@@ -0,0 +1,60 @@
+import util from 'util'
+import {
+  creatingErrMsg,
+  creatingOkMsg,
+} from '../../common/helpers/creatingResponse'
+import {
+  IPositiveResponse,
+  INegativeResponse,
+} from '../../common/interfaces/IResponse'
+import Logger from '@ioc:Adonis/Core/Logger'
+import TelegramBot from 'App/Services/Telegram'
+import Bnovo from 'App/Services/Bnovo'
+import { IPaidOrder } from '../../common/interfaces/IPaidOrder'
+
+export default class OrderService {
+  bnovo
+  telegramBot
```

Comment:
```suggestion
  private bnovo: Bnovo
  private telegramBot: Telegram
```
Либо можно указать модификатор видимости в параметрах конструктора и тогда вообще эти свойства не нужны.